skimage/transform/radon_transform.py

```diff
@@ -262,6 +266,20 @@ def iradon(radon_image, theta=None, output_size=None,
     if filter not in filter_types:
         raise ValueError("Unknown filter: %s" % filter)
 
+    if dtype is None:
+        if radon_image.dtype.char in 'efdg':
+            dtype = radon_image.dtype
+        else:
+            warn("Only floating point data type are valid for inverse radon "
+                 "transform. Input data is cast to float. To disable this "
+                 "warning, please cast image_radon to float.")
+            dtype = float
+    elif np.dtype(dtype).char not in 'efdg':
+        raise ValueError("Only floating point data type are valid for inverse "
+                         "radon transform.")
+
+    radon_image = radon_image.astype(dtype, copy=False)
```

Comment:
I think instead you should have a `preserve_range` argument and call `img_as_float(preserve_range)` or osmething like that. By default, preserve range would be `True` with a deprecation cycle bringing back to None.

Reply:
Is the deprecation cycle necessary? Wouldn't be `preserve_range` be directly set to `None` by default?